app/src/main/res/layout/edit_layer_dialog.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="horizontal"
+    android:paddingTop="@dimen/pad_1"
+    android:paddingBottom="@dimen/pad_1">
+
+    <TextView
+        android:id="@+id/editLayerName"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_weight="3" />
+
+    <ImageView
+        android:id="@+id/layerEditBtn"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_weight="1"
```

Comment:
These don't need weight. It's standard just to let icons be their normal size. You just need a weight of 1 on the textview.

Reply:
adjusted